indico/MaKaC/webinterface/pages/category.py

```diff
@@ -49,11 +47,30 @@
 
 from indico.core.index import Catalog
 from indico.modules import ModuleHolder
+from indico.modules.events.timetable.models.entries import TimetableEntryType
 from indico.modules.upcoming import WUpcomingEvents
 from indico.web.flask.util import url_for
 from indico.web.menu import render_sidemenu
 
 
+def format_location(obj):
+    if obj.inherit_location:
+        return ''
+
+    room_name = obj.room_name
+    if obj.room:
+        if obj.room.map_url:
```

Comment:
Great idea, ends up looking pretty nice if we use the one we've already got for events! :+1: